packages/pubsub-floodsub/test/floodsub.spec.ts

```diff
@@ -1,7 +1,7 @@
 /* eslint-env mocha */
 
 import { generateKeyPair } from '@libp2p/crypto/keys'
-import { type Message, type PubSubRPC, StrictNoSign } from '@libp2p/interface'
+import { type Message, type PubSubRPC, StrictNoSign, isPubSub } from '@libp2p/interface'
```

Comment:
this import is unused
```suggestion
import { type Message, type PubSubRPC, StrictNoSign } from '@libp2p/interface'
```